### PR TITLE
add logging for cases where the GlobalRegistry does not start

### PR DESCRIPTION
**Pull request by copperlight**

### CODE REVIEW

File: spectator/__init__.py [hunk 1]
```diff
@@ -7,10 +8,15 @@
...
 try:
     from spectatorconfig import auto_start_global
+
     if auto_start_global():
         GlobalRegistry.start()
         atexit.register(GlobalRegistry.stop)
-except:
-    pass
+    else:
+        logger.warning("module spectatorconfig auto-start is disabled - GlobalRegistry will not start")
```
Comment (brharrington) [inline spectator/__init__.py]: I would make this debug or info at the highest. If auto-start is legitimately disabled, then the user shouldn't get a regular warning.

File: spectator/__init__.py [hunk 2]
```diff
@@ -7,10 +8,15 @@
...
+except ImportError:
+    logger.error("module spectatorconfig is missing - GlobalRegistry will not start")
```
Comment (brharrington) [inline spectator/__init__.py]: The downside to this is that for things like running tests or other uses where you do not need it you will now get an error logged.

Comment (copperlight) [reply spectator/__init__.py]: What about moving it to debug level?

It's really common for people to forget to add the bindings and wonder why nothing works.

File: spectator/__init__.py (hunk 1 above)
Comment (copperlight) [reply spectator/__init__.py]: Agree with moving it to debug.

**Pull request merged**